CMakeLists.txt

```diff
@@ -69,7 +69,7 @@ Cabana_add_dependency( PACKAGE MPI )
 Cabana_add_dependency( PACKAGE ArborX )
 
 # find Clang Format
-Cabana_add_dependency( PACKAGE CLANG_FORMAT )
+find_package( CLANG_FORMAT )
```

Comment:
Does this imply that this is now a requirement?

Reply:
No. The clang_format just doesn't need our magic to create an `ENABLE_*` options, plus one could even see anymore what version was used, but we could certainly undo this change.